sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/aio/_schema_registry_client_async.py

```diff
@@ -170,6 +170,34 @@ async def get_schema(self, schema_id: str, **kwargs: Any) -> Schema:
         response.raise_for_status()
         return _parse_response_schema(response)
 
+    async def get_schema_by_version(self, group_name: str, name: str, version: int, **kwargs: Any) -> Schema:
+        """
+        Gets a registered schema within the specified schema group, given its name and version.
```

Comment:
Nit: clear rewording:
```suggestion
        Gets a specific version of a schema within the specified schema group.
```